src/components/shortcutsContainer.html

```diff
@@ -26,6 +26,13 @@
               </button>
             </td>
           </tr>
+          <tr id="dockShortcut" data-bind="visible: Settings.getSetting('enableDockShortcut').value">
+            <td class="p-0">
+              <button class="btn btn-block btn-primary m-0" onclick="MapHelper.openShipModal()">
+                  Docks
+              </button>
+            </td>
+          </tr>
```

Comment:
I've mentioned previously in the Discord, the Dock shortcut is not something to be added to the game.

Reply:
Totally out of the question even it's opt-in within Settings and disabled by default? I thought that was a good compromise. I'm thinking moreso along the line of mobile compatability here, as smaller screens I can actually understand a dock might be annoying to scroll up to or find.

If you say no in the end, do I revert these commits or would you be able to pick and choose your end, not used GitHub too much. I don't mind either way, btw, just thought i'd get your opinion of an actual commit to see how it would work.

Reply:
Unfortunately you can't just pick and choose which commits to merge in a PR, the whole PR is merged at once, so will need to be reverted on your end.

Reply:
Gotcha, no problem. So just to clarify, sorry for being a pest, that's definite no? 
No issue to revert at all, all seperate commits. :)